MeshLib/MeshGenerators/RasterToMesh.cpp

```diff
@@ -40,208 +44,194 @@ MeshLib::Mesh* RasterToMesh::convert(
     UseIntensityAs intensity_type,
     std::string const& array_name)
 {
-    if ((elem_type != MeshElemType::TRIANGLE) && (elem_type != MeshElemType::QUAD))
+    if ((elem_type != MeshElemType::TRIANGLE) &&
+        (elem_type != MeshElemType::QUAD) &&
+        (elem_type != MeshElemType::HEXAHEDRON) &&
+        (elem_type != MeshElemType::PRISM))
     {
-        ERR("VtkMeshConverter::convertImgToMesh(): Invalid Mesh Element Type.");
+        ERR("Invalid Mesh Element Type.");
         return nullptr;
     }
 
-    vtkSmartPointer<vtkDataArray> pixelData = vtkSmartPointer<vtkDataArray>(img->GetPointData()->GetScalars());
     int* dims = img->GetDimensions();
+    if (((elem_type == MeshElemType::TRIANGLE) || (elem_type == MeshElemType::QUAD)) &&
+        dims[2] != 1)
+    {
+        ERR("Triangle or Quad elements cannot be used to construct meshes from 3D rasters.");
+        return nullptr;
+    }
+
+    vtkSmartPointer<vtkDataArray> pixelData = vtkSmartPointer<vtkDataArray>(img->GetPointData()->GetScalars());
```

Comment:
Formatting could be improved.

Reply:
How?

Reply:
I'm not sure if you can use clang format under windows.
```
vtkSmartPointer<vtkDataArray> pixelData =
     vtkSmartPointer<vtkDataArray>(img->GetPointData()->GetScalars());
```

Reply:
I *did*, in fact, use clang-format. It didn't do anything here. But I can change it...